requirements.txt

```
@@ -1 +1,3 @@
-ops==2.0.0
\ No newline at end of file
+ops==2.0.0
+jinja2
+toml
```

Would be good to pin the versions of jinja and toml so that we know what versions we are specifically pulling. Also, I  think we can get away with pinning ops to 2.*:

```suggestion
ops == 2.*
jinja2
toml
```